antiphoneclapper/__init__.py

```diff
@@ -1,5 +1,7 @@
 from .antiphoneclapper import AntiPhoneClapper
 
+__red_end_user_data_statemet__ = "This cog does not persistently store data or metadata about users."

```

Comment:
did you mean ... __red_end_user_data_statement__ ? also this applies to every use of this. not like it's breaking things but would be nice to have it spelled right